TACC/Core-Portal
# bug/FP-1169 - Fix Paginator component

## Overview: ##
Fix paginator component to display correct number of pages in onboarding admin.

## Related Jira tickets: ##

* [FP-1169](https://jira.tacc.utexas.edu/browse/FP-1169)

## Summary of Changes: ##
Added clause to paginator if/else tree to account for 2-page paginators.
## Testing Steps: ##
1. Add the contents of the attached txt file to a fixture.js file named `OnboardingAdminFixtureExtended.fixture.js` in `client/src/components/Onboarding/`.
2. Add `import { onboardingAdminFixtureExtended } from './OnboardingAdminFixtureExtended.fixture';` to `OnboardingAdmin.js`.
3. Add `useSelector(state => state.onboarding.admin = onboardingAdminFixtureExtended` at line 261 in `OnboardingAdmin.js`.
4. Navigate to your dev portal and go to 'workbench/onboarding/admin'.
5. Verify paginator at bottom of page shows a box for page 1 and one for page 2.

[onboardingAdmixExtendedFixture.txt](https://github.com/TACC/Core-Portal/files/7198469/onboardingAdmixExtendedFixture.txt)


## UI Photos:
![image](https://user-images.githubusercontent.com/43251554/133854466-6beff516-2745-4cb1-9914-952c693d8538.png)


## Notes: ##
The included fixture does not include the full onboarding state, which seems to limit functionality of the paginator, though it should still render correctly. Please let me know if you would like this fixture to include the full onboarding portion of state and I will attempt to provide those further attributes.
